_sass/base.scss

```diff
@@ -88,14 +88,18 @@ footer {
 /* Layout */
 
 .container {
-  display: flex;
+  // display: flex;
```

Comment:
Don't commit commented out code! Please delete this.

Reply:
https://gph.is/1wQxdeR

Reply:
https://media.giphy.com/media/n2eijGHQirF3q/giphy.gif